frontend/src/app/testResults/TestResultsList.tsx

```diff
@@ -557,135 +529,180 @@ export const resultsCountQuery = gql`
   }
 `;
 
-type OmittedProps =
-  | InjectedQueryWrapperProps
-  | "pageCount"
-  | "entriesPerPage"
-  | "totalEntries"
-  | "loadingTotalResults"
-  | "facilityId"
-  | "setSelectedPatientId"
-  | "setResultFilter"
-  | "resultFilter"
-  | "setRoleFilter"
-  | "roleFilter"
-  | "setStartDateFilter"
-  | "startDateFilter"
-  | "setEndDateFilter"
-  | "endDateFilter";
-
-type TestResultsListProps = Omit<Props, OmittedProps>;
+export const testResultQuery = gql`
```

Comment:
**question:** Did this query change, at all? Or just move down in the file

Reply:
just moved it down for better organization